techniques/system/common/1.0/update.cf

```diff
@@ -25,11 +24,9 @@
 # Use for sensitive files
 body copy_from remote(server, path)
 {
-
         servers    => {
           "${server}"
         };
-        encrypt    => "true";
```

Comment:
Is that the default ?

Reply:
Good spot it is not actually. This has practically no impact as "When used with protocol_version 2 or greater this attribute is a noop as the entire session is encrypted." but who knows, it's probaly better to keep it anyway.